src/zimlet/com_zimbra_tos/com_zimbra_tos.properties

```diff
@@ -0,0 +1,24 @@
+#
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Zimlets
+# Copyright (C) 2010, 2013, 2014, 2016, 2018 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+
+label = Terms of Services
+description = Displays Terms of Services
+
+TosZimlet_label = Terms of Services
+TosZimlet_description = You must agree to the Terms of Service before you can proceed
+
+
+TosZimlet_content = Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis metus nisl, fringilla non molestie vel, laoreet id lectus. Fusce efficitur luctus nulla. Nam quis pellentesque enim. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer fermentum urna at ipsum placerat, ac laoreet leo gravida. Pellentesque vehicula, velit ac auctor ultricies, mauris sem imperdiet mauris, ut vulputate quam velit ut nisl. Quisque orci ante, rutrum a magna et, sodales sagittis sem.<br/><br/> \n Nulla rutrum mi et mi lobortis congue.Nulla suscipit placerat elementum.Duis sit amet mi tortor.Duis at ligula eget ligula faucibus egestas eu eu purus.Integer et sagittis tellus.Praesent dapibus sagittis diam, nec semper ligula bibendum eget.Vestibulum laoreet, odio vel fermentum elementum, tellus mi hendrerit nulla, sit amet semper ligula odio vitae diam.Cras efficitur tempor velit, id posuere nisl scelerisque vel.Integer ac arcu convallis, commodo lorem ut, ultricies nibh.<br/><br/> \n Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis metus nisl, fringilla non molestie vel, laoreet id lectus. Fusce efficitur luctus nulla. Nam quis pellentesque enim. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer fermentum urna at ipsum placerat, ac laoreet leo gravida. Pellentesque vehicula, velit ac auctor ultricies, mauris sem imperdiet mauris, ut vulputate quam velit ut nisl. Quisque orci ante, rutrum a magna et, sodales sagittis sem.
```

Comment:
multi-line format is easiler to read or manage.

    TosZimlet_content = Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis metus nisl, fringilla non molestie vel, laoreet id lectus. \
       Fusce efficitur luctus nulla. Nam quis pellentesque enim. Lorem ipsum dolor sit amet, consectetur adipiscing elit. \
       Integer fermentum urna at ipsum placerat, ac laoreet leo gravida.  ....

You can see the similar definition in com_zimbra_attachmentalert_it.properties or ZmMsg.properties.
Please be careful not removing necessary whitespace if you change the format.